MekHQ/src/mekhq/gui/adapter/UnitTableMouseAdapter.java

```diff
@@ -379,17 +379,9 @@ public void actionPerformed(ActionEvent action) {
             ((MekLabTab) gui.getTab(GuiTabType.MEKLAB)).loadUnit(selectedUnit);
             gui.getTabMain().setSelectedIndex(GuiTabType.MEKLAB.getDefaultPos());
         } else if (command.equals(COMMAND_CANCEL_CUSTOMIZE)) {
-            for (Unit unit : units) {
-                if (unit.isRefitting()) {
-                    selectedUnit.getRefit().cancel();
-                }
-            }
+            Stream.of(units).filter(Unit::isRefitting).forEach(unit -> unit.getRefit().cancel());
```

Comment:
Is this fix essentially that `Stream.of` skips null lists?

Reply:
The problem is that selected unit is used instead of the unit... so it NPEs on the selectedUnit.getRefit() call after the first unit.

Reply:
Hah, missed that.